examples/cryptorealtime/README.md

```diff
@@ -0,0 +1,159 @@
+
+## Project setup 
+### Install the Google Cloud Platform SDK on a new VM
+  * Log into the console, and activate a cloud console session
+  * Create a new VM
+```console
+gcloud beta compute instances create crypto-driver \
+  --zone=us-central1-a \
+  --machine-type=n1-standard-1 \
+  --service-account=$(gcloud iam service-accounts list --format='value(email)' --filter="compute") \
+  --scopes=https://www.googleapis.com/auth/cloud-platform \
+  --image=debian-9-stretch-v20181210 \
+  --image-project=debian-cloud \
+  --boot-disk-size=20GB \
+  --boot-disk-device-name=crypto-driver
+```
+
+
+  * SSH into that VM 
+
+```console
+gcloud compute ssh --zone=us-central1-a crypto-driver
+```
+  
+  * Installing necessary tools like java, git, maven, pip, python 2.7 and Cloud Bigtable command line tool cbt using the following command:
+```console
+apt -y install python2.7 python-pip openjdk-8-jdk git maven google-cloud-sdk-cbt
+```
+
+### Create a Google Cloud Bigtable instance 
+```console
+export PROJECT=$(gcloud info --format='value(config.project)')
+export ZONE=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/zone" -H "Metadata-Flavor: Google"|cut -d/ -f4)
+gcloud services enable bigtable.googleapis.com \
+bigtableadmin.googleapis.com \
+dataflow.googleapis.com 
+
+gcloud bigtable instances create cryptorealtime \
+  --cluster=cryptorealtime-c1 \
+  --cluster-zone=${ZONE} \
+  --display-name=cryptorealtime \
+  --cluster-storage-type=HDD \
+  --instance-type=DEVELOPMENT
+cbt -instance=cryptorealtime createtable cryptorealtime families=market
+```
+
+### Create a Bucket  
+```console 
+gsutil mb -p ${PROJECT} gs://realtimecrypto-${PROJECT}
+```
+
+### Create firewall for visualization server on port 5000
+```console 
+gcloud compute firewall-rules create crypto-dashboard --action=ALLOW --rules=tcp:5000 --source-ranges=0.0.0.0/0 --target-tags=crypto-console --description="Open port 5000 for crypto visualization tutorial"
+  
+gcloud compute instances add-tags crypto-driver --tags="crypto-console" --zone=${ZONE}
+```
+  
+
+### Clone the repo
+```console 
+git clone https://github.com/GoogleCloudPlatform/professional-services
+```
+
+### Build the pipeline
+```console 
+cd professional-services/examples/cryptorealtime
+mvn clean install
+```
+
+### Start the DataFlow pipeline
+```console 
+./run.sh ${PROJECT} \
+cryptorealtime gs://realtimecrypto-${PROJECT}/temp \
+cryptorealtime market
+``` 
+
+### Start the Webserver and Visualization
+```console 
+cd frontend/
+pip install -r requirements.txt --user
+python app.py ${PROJECT} cryptorealtime cryptorealtime market
+```
+
+Find your external IP in [Compute console instance](https://console.cloud.google.com/compute/instances) and open it in your browser with port 5000 at the end e.g.
+http://external-ip:5000/stream
+
+You should be able to see the visualization of aggregated BTC/USD pair on several exchanges (without predictor part)
+
+
+#Cleanup
```

Comment:
Add space after `#` so the heading is rendered correctly.

Reply:
8af3520